setup/05-global-tables/movr/pricing/data/pricing_database.sql

```diff
@@ -0,0 +1,13 @@
+CREATE DATABASE movr_pricing;
```

Comment:
init_db.sh doesn't run this file, so the pricing db is not created

Reply:
included in init file